posts/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,600 @@
+
+
+* link:{url-prefix}/docs/latest/reference/javadoc/liberty-jakartaee10-javadoc.html[Jakarta EE 10 Javadoc]
+
+
+
+   
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/24582
+// Contact/Reviewer: ReeceNana,Emily-Jiang
+// // // // // // // // 
+[#mp6]
+== MicroProfile 6.0   
+
+MicroProfile continues to innovate how the industry optimizes Java microservices.  The MicroProfile 6.0 release enables applications to use MicroProfile APIs together with <<core, Jakarta EE Core Profile 10>> along with various other new functionality and improvements. The complete list of specifications that make up MicroProfile 6.0 is as follows:
+
+
+[cols="4,2,3"]
+|===
+|Specification |Updates |Liberty Feature Documentation
+
+[[telemetry]]
+|https://github.com/eclipse/microprofile-telemetry/releases/tag/1.0[MicroProfile Telemetry 1.0]
+|*New*
+|link:{url-prefix}/docs/latest/reference/feature/mpTelemetry-1.0.html[mpTelemetry-1.0]
+
+[[metrics]]
+|https://github.com/eclipse/microprofile-metrics/releases/tag/5.0[MicroProfile Metrics 5.0]
+|Major update
+|link:{url-prefix}/docs/latest/reference/feature/mpMetrics-5.0.html[mpMetrics-5.0]
+
+[[openapi]]
+|https://github.com/eclipse/microprofile-open-api/releases/tag/3.1[MicroProfile OpenAPI 3.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/mpOpenAPI-3.1.html[mpOpenAPI-3.1]
+
+[[jwt]]
+|https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.1[MicroProfile JWT Authentication 2.1]
+|Minor update
+|link:{url-prefix}/docs/latest/reference/feature/mpJwt-2.1.html[mpJwt-2.1]
+
+|https://github.com/eclipse/microprofile-config/releases/tag/3.0.2[MicroProfile Config 3.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpConfig-3.0.html[mpConfig-3.0]
+
+|https://github.com/eclipse/microprofile-health/releases/tag/4.0.1[MicroProfile Health 4.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpHealth-4.0.html[mpHealth-4.0]
+
+|https://github.com/eclipse/microprofile-rest-client/releases/tag/3.0.1[MicroProfile Rest Client 3.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpRestClient-3.0.html[mpRestClient-3.0]
+
+|https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/4.0.2[MicroProfile Fault Tolerance 4.0]
+|Unchanged
+|link:{url-prefix}/docs/latest/reference/feature/mpFaultTolerance-4.0.html[mpFaultTolerance-4.0]
+
+|https://jakarta.ee/specifications/coreprofile/10/[Jakarta EE Core Profile 10]
+
+|New
+|<<coreprofile, See Core Profile table>>
+
+
+|===
+
+You can enable the MicroProfile 6 feature by using the `microProfile-6.0` feature in the server.xml file:

```

Comment:
Style thing: just use the same format of lead-in sentence for each server.xml instruction. Don't vary the wording each time; just keep it simple and recognisable so that it's clear that nothing is different except the code that you add.